setup.py

```diff
@@ -20,6 +20,11 @@
 from shutil import copy
 
 try:
+    import setuptools
+except ImportError:
+    pass  # setuptools allows for "develop" and entry_points, but it's not essential
```

Comment:
Good point. Since we implement `develop` ourselves, it probably makes sense to restrict it yes.